EIPS/eip-4750.md

```diff
@@ -32,23 +32,14 @@ Furthermore, it aims to improve analysis opportunities by encoding the number of
 1. The requirement of [EIP-3540](./eip-3540.md) "Exactly one code section MUST be present." is relaxed to "At least one code section MUST be present.", i.e. multiple code sections (`kind = 1`) are allowed.
 2. Total number of code sections MUST NOT exceed 1024.
 3. All code sections MUST precede a data section, if data section is present.
-4. New section with `kind = 3` is introduced called the *type section*.
+4. New section with `kind = 1` is introduced called the *type section*.
```

Comment:
Type section is now of same type than code section..?

Reply:
No code section kind now is `2`, see changes to EIP-3540